smoke-tests/src/test/java/io/opentelemetry/smoketest/windows/WindowsTestContainerManager.java

```diff
@@ -314,6 +314,8 @@ private Container startContainer(
 
     CreateContainerCmd createCommand = client.createContainerCmd(imageName);
     createAction.accept(createCommand);
+    // default appears to be 1gb which is not enough for liberty
+    createCommand.getHostConfig().withMemory(2_000_000_000L);
```

Comment:
Is the problem that jvm ergonomics isn't able to figure out that it is running inside container and picks way too large heap size? Maybe we should add something like `-Xmx512m` or less in `AbstractTestContainerManager.getAgentEnvironment` where we also add `-javaagent`

Reply:
good call

`java -XX:+PrintFlagsFinal -version | grep -w MaxHeapSize`

* linux container with 1g memory limit: `268,435,456`
* linux container with "default gha" memory limit: `1,824,522,240`
* windows container with 1g memory limit: `1,879,048,192`
* windows container with "default gha" memory limit: `1,879,048,192`

https://github.com/trask/test-jvm-container-ergonomics/actions/runs/1156983349